lib/settings.ts

```diff
@@ -91,19 +91,39 @@ const readConfigFile = (file: string): object => {
 };
 
 const replaceResinKeys = (parsedConfig: object) =>
-	_.mapKeys(parsedConfig, (_value, key) => key.replace('resin', 'balena'));
+	Object.fromEntries(
+		Object.entries(parsedConfig).map(([key, value]) => [
+			key.replace('resin', 'balena'),
+			value,
+		]),
+	);
 
-const getSettings = _.once((): { [k: string]: string | undefined } =>
-	utils.mergeObjects(
+// Helper to safely transform if the file exists
+const safeLoad = (path: string, transform?: (obj: any) => any) => {
```

Comment:
"safeLoad" sounds a bit too re-assuring for just avoiding nulls tbh. I also don't see any loading happening in here.

I guess you could completely avoid by just adding  and `if (parsedConfig == null) { return parsedConfig; }` in `replaceResinKeys`.
If you still prefer keeping this, then how about calling it `pipeTransform` or something?